spec/factories/wishlists.rb

```diff
@@ -0,0 +1,12 @@
+FactoryGirl.define do
+  factory :wishlist do
+    name "Birthday"
+    description "Tom's 18th birthday"
+  end
```

Comment:
Fill the association? https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#associations